conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -158,6 +158,7 @@ types:
           docs: optional<Documentation>
           deprecated: optional<Documentation>
           markers: list<Type>
+          tags: set<string>
```

Comment:
`docs` uses the `Documentation` `alias<string>` type, should we use a string alias here as well?

Reply:
Is there a reason we don't use an enum if we're defining a new field? It would be nice for this to be more strongly typed, unless the assertion is that different generators might respect specific/different/new/custom tags.

Reply:
Idea is that it could include some kind of SLO information with a dynamic component, so it's not clear that constant values will be terribly useful.

Reply:
Adding an enum would suggest to folks that there are only a fixed set of valid options for the field instead of goal which is to give developers (and ourselves) more flexibility in the metadata they can attach to a conjure definitions.